Team9Project/src/com/indragie/cmput301as1/ExpenseClaimDetailActivity.java

```diff
@@ -151,6 +160,22 @@ public void onDateChanged(DateEditText view, Date date) {
 				startDateField.setMaxDate(date);
 			}
 		});
+		
+		userField = (TextView)headerView.findViewById(R.id.tv_user);
+		userField.append(claim.getUser().getName());
+		
+		approverField = (TextView)headerView.findViewById(R.id.tv_approver);
+		try{
```

Comment:
Will throw a null exception when the claim is first created, no approval user has been added to it until it is either returned or approved.